jupyter-book/glossary.md

```diff
@@ -1,41 +1,56 @@
 # Glossary
 
 ```{glossary}
+Adapter sequences
+adapter sequences
+    Short, synthetic DNA or RNA sequences that are ligated to the ends of DNA or RNA fragments during library preparation for sequencing.
+    These adapters are essential for binding the fragments to the flowcell and enabling amplification and sequencing.
+    However, if adapters are not properly removed or trimmed after sequencing, they can appear in the reads, potentially interfering with alignment and downstream analyses.
+
 Algorithm
 Algorithms
     A pre-defined set of instructions to solve a problem.
 
 AnnData
 AnnDatas
-    A Python package for annotated data matrices. The primary data structure used in the scverse ecosystem.
+    A Python package for annotated data matrices.
+    The primary data structure used in the scverse ecosystem.
+
+Backtrace
```

Comment:
Not sure whether we need this glossary item. Looks like bioinformatics course PTSD though hahhaaha